client/my-sites/site-settings/amp/wpcom.jsx

```diff
@@ -32,7 +33,8 @@ class AmpWpcom extends Component {
 	};
 
 	handleToggle = () => {
-		const { fields, submitForm, trackEvent, updateFields } = this.props;
+		const { fields, submitForm, trackEvent, updateFields, trackTracksEvent } = this.props;
+		trackTracksEvent( 'calypso_seo_settings_amp_updated', { amp_is_enabled: ! fields.amp_is_enabled } );
```

Comment:
I'd store `{ amp_is_enabled: ! fields.amp_is_enabled }` in a variable above, then use it here and in the `updateFIelds` call for 2 reasons:

* To avoid repetition (stay DRY)
* To reduce the possibility for errors if we decide to change this and change only one of the occurrences.